plugins/CorePluginsAdmin/Commands/DeactivatePlugin.php

```diff
@@ -38,6 +38,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 continue;
             }
 
+            if (!$pluginManager->isPluginInFilesystem($plugin)) {
+                $output->writeln("<error>Cannot find plugin files for $plugin.</error>");
+                continue;
+            }
```

Comment:
Does that check make sense here?
If a plugin name is provided that was never installed (is not included in plugin list in config.ini.php), the method will already fail with `already deactivated`. If it was installed, but only removed from file system it might make sense to allow a deactivation so it gets removed from the plugin list in config.ini.php and it's no longer tried to load it anymore.

Reply:
:+1: makes sense to me